### PR TITLE
Barcode implementation

### PR DESCRIPTION
**Pull request by shepkeira**
- I can't get scanning to work because its weird with docker
- Also to my understanding it will actually differ between us because we are using different types of computers (mac vs windows vs linux) since the ports are different
- I've decided we can probably ignore that and just upload an image, that will make peer testing easier anyways

### CODE REVIEW

File: LIMS_IMAGE/web/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+RUN apt-get update -y
+RUN apt-get install zbar-tools -y
 RUN apt-get update && apt-get install -y --no-install-recommends \
     unixodbc-dev \
     unixodbc \
     libpq-dev \
     curl
 ADD requirements.txt requirements.txt
 RUN pip install -r requirements.txt
+RUN pip install -U matplotlib
```
Comment (Rock-it-science) [inline LIMS_IMAGE/web/Dockerfile:13]: Would this be able to go into requirements.txt?

**Review by Rock-it-science — APPROVED** [review]
Looks really good! That's too bad about the scanner thing, but considering we don't have scanners anyways this will suffice I'm sure, and you're right it will make peer testing easier. Nice work.

File: LIMS_IMAGE/web/Dockerfile (hunk 1 above)
Comment (shepkeira) [reply LIMS_IMAGE/web/Dockerfile:13]: No :( because its -U and not -r it doesn't work. I did try it though

Comment (Rock-it-science) [reply LIMS_IMAGE/web/Dockerfile:13]: Okay just checking. Should be all good then

**Pull request merged**